### DISCUSSION TITLE
public/nestedDir/index.html - default index - without typing index.html

### DISCUSSION THREAD

**Discussion by petar-barzev-cko** [vercel/next.js#14739] [Help]
Hi,

I am trying to serve a static website inside `public/nestedDir/`. Inside `nestedDir` I have an `index.html` file.

Now, I can get to the file if I do `localhost:3000/nestedDir/index.html`, but how can I make index available as a default file type so I can just hit `localhost:3000/nestedDir` and open the index file (no need to specify it).

Another approach I've tried is: setup an NGINX in sidecar pattern when I run in container mode - that is how I will be accessing this static website. So I've also been to use an NGINX rule to rewrite `/nestedDir/` to `/nestedDir/index.html`  via a `location` block, but haven't been able to do so successfully. I would then still need to do `localhost/nestedDir/index.html` to get to the page.

**Comment by rohmanhm** [comment]
To do that I believe you need a custom server setup. 
Here's a reference if you wanna use express server https://github.com/vercel/next.js/tree/canary/examples/custom-server-express

**Comment by tdurand** [comment]
Is there a way to do this in a way it is compatible with vercel deployment ?
